doc/ctc-meetings/2016-06-29.md

```diff
@@ -0,0 +1,194 @@
+# Node Foundation CTC Meeting 2016-06-29
+
+## Links
+
+* **Audio Recording**: TBP
+* **GitHub Issue**: https://github.com/nodejs/node/issues/7474
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1a19l2ICQHCw2w6cM4AayRM8QX8cIrgnHXCtzmuxfWuM>
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1X9PTcpYgTKJO-PRKOLBrL4fIV-7XDR0x71FAxJWn0II>_
+
+## Present
+
+* Bradley Meck @bmeck (observer/GoDaddy/TC39)
+* Ben Noordhuis @bnoordhuis (CTC)
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+* Colin Ihrig @cjihrig (CTC)
+* Evan Lucas @evanlucas (CTC)
+* Jeremiah Senkpiel @Fishrock123 (CTC)
+* James M Snell @jasnell (CTC)
+* John-David Dalton @jdalton (observer/Lodash/Microsoft)
+* Josh Gavant @joshgav (observer/Microsoft)
+* Michael Dawson @mhdawson (CTC)
+* Julien Gilli @misterdjules (CTC)
+* Mikeal Rogers @mikeal (observer/Node.js Foundation)
```

Comment:
I don't think @mikeal was at this meeting. (That's the last one on the attendance nits!)